src/common/arrow/arrow_converter.cpp

```diff
@@ -177,11 +155,6 @@ void SetArrowFormat(DuckDBArrowSchemaHolder &root_holder, ArrowSchema &child, co
 		break;
 	}
 	case LogicalTypeId::VARCHAR:
-		if (type.IsJSONType() && options.arrow_lossless_conversion) {
```

Comment:
Where is this being handled now? I was expecting a `SetArrowExtension` call, similar to the `UUID` case above.

Reply:
On the top of the function there is an alias check
```if (type.HasAlias()) {
...
```